api/download.py

```diff
@@ -115,7 +115,7 @@ def _bulk_preflight_archivestream(self, file_refs):
                 file_cnt += 1
 
         if len(targets) > 0:
-            filename = arc_prefix + datetime.datetime.utcnow().strftime('%Y%m%d_%H%M%S') + '.tar'
+            filename = arc_prefix + '_ '+datetime.datetime.utcnow().strftime('%Y%m%d_%H%M%S') + '.tar'
```

Comment:
So it is in the same format as the [other download path](https://github.com/scitran/core/pull/446/files#diff-5f6b428f8ab6dc5134ff7b9cbcad2062R224).